lib/rand/random.go

```diff
@@ -26,30 +31,36 @@ var (
 	defaultSecureRand = mathRand.New(defaltSecureSource)
 )
 
-func init() {
-	// The default RNG should be seeded with something good.
-	mathRand.Seed(RandomInt64())
-}
```

Comment:
Indeed. I debated removing this, but we still use math/rand in a few places, and having it seeded by default somewhere is better than not doing it. We should do a cleanup later to not use math/rand anywhere, unless we need the predictable seeding or can show that we need the better performance of that package. And I don't think we ever need the performance as we're not generating that much random things